packages/core/src/shared/utilities/collectionUtils.ts

```diff
@@ -329,6 +330,31 @@ export function partialClone(obj: any, depth: number = 3, omitKeys: string[] = [
     return clonedObj
 }
 
+type inspectOptions = Partial<
+    nodeInspectOptions & {
+        omitKeys: string[]
+        replacement: any
```

Comment:
inspect doesn't need this options, because callers can easily use `inspect(partialClone(o))` to opt-in if they want (and that's a good general practice: composability instead of "denormalization").

The use of `partialClonse` for the "web" case in our `inspect` wrapper, is simply a way for us to support `depth`.